api/ontology/ontology_test.go

```diff
@@ -228,3 +228,23 @@ func TestProtoResource(t *testing.T) {
 		})
 	}
 }
+
+func TestGetResourceTypes(t *testing.T) {
+	tests := []struct {
+		name string
+		want assert.Want[[]string]
+	}{
+		{
+			name: "Happy path",
+			want: func(t *testing.T, got []string) bool {
+				return assert.NotEmpty(t, got)
```

Comment:
This test is ok, but maybe not the best (accurate).

Another option would be to hardcode the amount of resources and test this against the length of the list. We then would need to change it if we create new ontology objects. So we have to test each time if it still works properly.